easyDiffractionApp/Logic/PyQmlProxy.py

```diff
@@ -378,19 +381,50 @@ def projectInfoAsJsonSetter(self, json_str):
         self._project_info = json.loads(json_str)
         self.projectInfoChanged.emit()
 
+    @Property(str, notify=projectInfoChanged)
+    def projectInfoAsCif(self):
+        cif_list = []
+        for key, value in self.projectInfoAsJson.items():
+            if ' ' in value:
+                value = f"'{value}'"
+            cif_list.append(f'_{key} {value}')
```

Comment:
why create the distinction between keys with and without space? Just add quotes to all of them.

Reply:
This is done following the START/CIF format. In most cases quotation marks are unnecessary, so the large file becomes more readable.